### PR TITLE
add priorityclass

### PR DESCRIPTION
**Pull request by dza89**
**Why This PR?**
Add priorityclass option for all pods

Fixes #

**Changes**
Changes proposed in this pull request:

Priorityclass added

**Checklist:**

* [x ] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x ] Any new values are backwards compatible and/or have sensible default.
* [x ] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

**Comment by dza89** [comment]
Need some help with the check-docs, thought I fixed it but still error

**Review by sudermanjr — CHANGES_REQUESTED** [review]

File: stable/vpa/templates/admission-controller-deployment.yaml [hunk 1]
```diff
@@ -26,6 +26,9 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.priorityClassName }}
+      priorityClassName:
```
Comment (sudermanjr) [inline stable/vpa/templates/admission-controller-deployment.yaml]: I think you'll need to add the value too:

```suggestion
      priorityClassName: {{ . }}
```

**Review by sudermanjr — CHANGES_REQUESTED** [review]
One more version bump after pulling should take care of the helm-docs and helm-lint.

~Can you also add `priorityClassName: testPriority` to the ci/test-values.yaml file? This will make sure this value gets end-to-end tested.~ - Don't do that, it will fail the CI

**Review by sudermanjr — APPROVED** [review]
Thanks!

**Pull request merged**